front/tests/utils/teardown-utils.ts

```diff
@@ -0,0 +1,108 @@
+import type {
+  GetLightRollingStockApiResponse,
+  LightRollingStock,
+} from 'common/api/osrdEditoastApi';
+
+import {
+  getApiRequest,
+  deleteApiRequest,
+  getInfra,
+  getStudy,
+  getScenario,
+  getProject,
+} from './api-setup';
+
+/**
+ * Deletes infrastructure by name if it exists.
```

Comment:
```suggestion
 * Delete infrastructure by name if it exists.
```

to be consistent with every other functions in every utils, same for others `delete` in this file